src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -138,38 +138,47 @@ const connectGeoSearch = (renderFn, unmountFn) => {
     // Always trigger this message because the default value was `true`. We can't
     // display the message only when the parameter is defined otherwise a user that was
     // relying on the default value won't have any information about the changes.
-    warn(`
+    warning(
+      false,
+      `
 The option \`enableGeolocationWithIP\` has been removed from the GeoSearch widget.
 Please consider using the \`Configure\` widget instead:
 
 search.addWidget(
   configure({
-    aroundLatLngViaIP: ${widgetParams.enableGeolocationWithIP || 'true'},
+    aroundLatLngViaIP: ${(widgetParams &&
```

Comment:
The value `widgetParams` is always defined we can avoid the check on `widgetParams`.